.github/workflows/basic.yml

```
@@ -14,9 +14,17 @@ on:
         default: 45
         required: false
         type: number
+      working-directory:
+        description: The working directory where all jobs should be executed. useful if you've a module in a subdirectory (like a monorepo or a controlrepository)
```

I generally prefer the documentation to be impersonal
```suggestion
        description: The working directory where all jobs should be executed. Used for modules in subdirectories like a monorepo or a control repository.
```